### ISSUE TITLE
Quit visual mode when press `Escape`

### ISSUE THREAD

**Issue by usoonees** [vipzhicheng/logseq-plugin-vim-shortcuts#13]
After finishing the operation in visual mode, I am used to pressing `Escape` to quit visual mode, but currently I have to press the same key as the trigger.

**Comment by vipzhicheng** [comment]
Yes It's a good idea, I also want to implement it that way, but It's hard, I have given a suggestion to the Logseq team. When it is possible I will add it right away.
